source/reference/mongosqld.txt

```diff
@@ -680,6 +680,40 @@ Process Management Options
      - string
      - :option:`--serviceDescription <mongosqld --serviceDescription>`
 
+Set Parameter Options
+~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: sh
+
+   setParameter:
+     polymorphic_type_conversion_mode: <string>
+     type_conversion_mode: <string>
+
+.. list-table::
+   :header-rows: 1
+
+   * - Name
+     - Type
+     - Corresponds to
+
+   * - .. setting:: setParameter.polymorphic_type_conversion_mode
+     - string
+     - Determines how :binary:`~bin.mongosqld` evaluates document fields
+       that are specified with multiple data types. For a list of
+       accepted values, see
+       :ref:`System Variables <system-environment-variables>`.
+
+   * - .. setting:: setParameter.type_conversion_mode
+     - string
+     - Specifies the semantics that :binary:`~bin.mongosqld` uses for
+       type conversions, such as the CAST function. For a list of
+       accepted values, see
+       :ref:`System Variables <system-environment-variables>`.
```

Comment:
I would replace this with the block I changed earlier in this PR:
```
     - Specifies which mode the :binary:`~bin.mongosqld` uses to
       :doc:`convert data types </reference/type-conversion>`.
       |bi-short| uses its own mode (``mongosql``) by default. Accepted
       values are ``mongosql`` and ``mysql``.
```
There were only two values, so just listing them here made sense.